src/core/Mixer.cpp

```diff
@@ -192,9 +192,9 @@ Mixer::Mixer() :
 	JournallingObject(),
 	m_mixerChannels()
 {
+	m_lastSoloed = -1;
```

Comment:
Since you moved this line above `createChannel`, can you move this to the initializer list of this constructor as well?

Reply:
Done